src/BIP32.php

```diff
@@ -161,7 +161,7 @@ public static function CKD($master, $address_definition, $generated = array())
         }
 
         if (!isset($data)) {
-            return false;
+            throw new \Exception(); // I don't think this can happen
```

Comment:
It depends on is_prime_hex. $data should always be set, you can remove this! Almost tempted to say the same for L147